portfolio/src/main/webapp/contactMeForm.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Contact Me Form</title>
+  </head>
+  <body class="body">
+    <h1>hmu!</h1>
+    <form href="/submitForm">
+      <input type="text" id="name" value="name" name="name" />
+      <br />
+      <input type="text" id="email" value="email" name="email" />
+      <br />
+      <input type="text" id="reason" value="reason" name="reason" />
```

Comment:
Tip: consider using the [`placeholder`](https://www.w3schools.com/tags/att_input_placeholder.asp) attribute instead of the `value` attribute. Usually you don't want to set a `value` unless there's a reasonable default.